src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/npm-shrinkwrap.json

```diff
@@ -1184,9 +1184,9 @@
       "resolved": "https://registry.npmjs.org/isomorphic-fetch/-/isomorphic-fetch-2.2.1.tgz"
     },
     "jquery": {
-      "version": "3.2.1",
-      "from": "jquery@>=3.2.1 <4.0.0",
-      "resolved": "https://registry.npmjs.org/jquery/-/jquery-3.2.1.tgz"
+      "version": "3.3.1",
+      "from": "jquery@>=3.3.1 <4.0.0",
+      "resolved": "https://registry.npmjs.org/jquery/-/jquery-3.3.1.tgz"
```

Comment:
Normally the `npm-shrinkwrap.json` files (which BTW will change to `package-lock.json` in the next update) are updated by running NPM, e.g., `npm install --save jquery@^3.3.1`. The change you've done manually here is probably fine in this case though, since this version of `npm-shrinkwrap.json` doesn't specify a SHA for the content.

Reply:
When I tried the npm install command, it completely changed the npm-shrinkwrap.json file. For instance, removing `from` and adding other fields.

Reply:
Running the above command correctly updates the `package.json` file but deletes all content from the `npm-shrinkwrap.json` file except the jQuery node. Is doing this manually fine then?